### PR TITLE
i3 installation from testing repository

### PR DESCRIPTION
**Pull request by minad**
See https://github.com/QubesOS/qubes-desktop-linux-i3/pull/1

### CODE REVIEW

**Comment by minad** [comment]
ping @marmarek

File: customization/i3.md [hunk 1]
```diff
@@ -9,19 +9,27 @@ redirect_from:
...
+
+## Compilation and installation from source
+
+Note that the compilation from source is done in a Fedora based domU (could
 be dispvm). The end result is always an `.rpm` that is copied to dom0 and then
 installed through the package manager.
 
-**Warning:** Installing software in dom0 is inherently risky, and the method
-described in this guide circumvents the usual security mechanisms for [updating
-software in dom0](/doc/software-update-dom0/).
```
Comment (marmarek) [inline customization/i3.md]: I'd not remove this warning, as the procedure below does not involve any source code verification.

Comment (marmarek) [inline customization/i3.md]: Or, instead of this warning, put another one about lack of source verification.

**Comment by minad** [comment]
I just moved the warning a bit down to the compilation from source section. The installation from testing is verified I guess?

File: customization/i3.md [hunk 2]
```diff
@@ -50,8 +57,9 @@ you get the sources and start off the build:
 
     $ make rpms
 
-Installing
-----------
+### Installing
+
+**Warning**: Manually installing software in dom0 is inherently risky, and the method described here circumvents the usual security mechanisms of qubes-dom0-update.
```
Comment (minad) [inline customization/i3.md]: warning from above

File: customization/i3.md [hunk 3]
```diff
@@ -74,8 +82,9 @@ After that you can just install the generated rpm like any other local package
 
 Log out, select i3, then log in again.
 
-Configuration
--------------
+### Configuration
+
+**Warning**: Be careful when writing configuration/scripts for the dom0. A script which communicates with the qubes VMs could potentially open a security hole.
```
Comment (minad) [inline customization/i3.md]: additional warning about writing config scripts

**Pull request merged**